README.md

```diff
@@ -67,6 +78,14 @@ With `current` or `all` you can change gaps either for only the currently focuse
 
 You can find an example configuration in the [wiki](https://github.com/Airblader/i3/wiki/Example-Configuration).
 
+### Rounded Window Corners
+
+Rounded corners can be configured by adding this to your config. `5` can be replaced with any integer, it simply defines how rounded the corners are.
```

Comment:
NIT: "how rounded" could be more specifically defined as the radius of the rounded corner, but since the units are made up anyways it doesn't really change anything